video/out/wayland_common.c

```diff
@@ -1434,8 +1434,10 @@ static void registry_handle_add(void *data, struct wl_registry *reg, uint32_t id
 #else
         ver = MPMIN(ver, 7);
 #endif
-        wl->seat = wl_registry_bind(reg, id, &wl_seat_interface, ver);
-        wl_seat_add_listener(wl->seat, &seat_listener, wl);
+        if (!wl->seat) {
+            wl->seat = wl_registry_bind(reg, id, &wl_seat_interface, ver);
+            wl_seat_add_listener(wl->seat, &seat_listener, wl);
+        }
```

Comment:
You can drop this commit since the rest of the PR completely rewrites it anyway.